src/telepathy/Makefile.am

```diff
@@ -35,6 +35,8 @@ telepathy_sipe_CFLAGS = \
 
 telepathy_sipe_LDADD = \
 	../core/libsipe_core.la \
+	../purple/libsipe_backend_vv.la \
+	../purple/libsipe_backend.la \
```

Comment:
Oh, sure. I didn't actually look what you've done here. These two have no business being used with Telepathy backend and are the reason for multiple definition during linking. Why did you add them?